### PR TITLE
3.36.1: Adds new SDK versions and contract files

### PR DESCRIPTION
**Pull request by kundadebdatta**
# Pull Request Template

## Description

This bumps the SDK version from `3.36.0` to `3.36.1`. Please see the below changes in the GA public contract.

image

Also highlighted below are the preview contract changes.

image

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -13,6 +13,17 @@ Preview features are treated as a separate branch and will not be included in th
...
 
+### <a name="3.36.1-preview"/> [3.36.1-preview](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.36.1-preview) - 2023-11-17
+### <a name="3.36.1"/> [3.36.1](https://www.nuget.org/packages/Microsoft.Azure.Cosmos/3.36.1) - 2023-11-17
+
+#### Fixed
+- [4100](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4100) Query : Fixes querying conflicts (#4100)
+- [4125](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4125) Item Operations: Fixes JsonSerialization exception when MissingMemberHandling = Error on Json default settings when NotFound on Item operations (#4125)
+
+#### Added
+- [4180](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4180) Upgrade Resiliency: Adds Code to Enable Advanced Replica Selection Feature for Preview (#4180)
```
Comment (sourabh1007) [inline changelog.md:24]: If this is applicable for preview package then it should go under that.

Comment (kundadebdatta) [reply changelog.md:24]: This is applicable for GA as well. I will update the verbiage. The PR title and description covers this.

File: changelog.md [hunk 2]
```diff
@@ -13,6 +13,17 @@ Preview features are treated as a separate branch and will not be included in th
...
+- [4128](https://github.com/Azure/azure-cosmos-dotnet-v3/pull/4128) Routing: Adds ExcludeRegions Feature to RequestOptions (#4128)
```
Comment (sourabh1007) [inline changelog.md:25]: As a practice, should we make public contract changes in minor release? cc @kirankumarkolli @ealsur

File: changelog.md (hunk 1 above)
Comment (sourabh1007) [reply changelog.md:24]: changing it in changelog should be enough. No need to change PR title :)

File: Directory.Build.props [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<PropertyGroup>
-		<ClientOfficialVersion>3.36.0</ClientOfficialVersion>
-		<ClientPreviewVersion>3.36.0</ClientPreviewVersion>
+		<ClientOfficialVersion>3.36.1</ClientOfficialVersion>
```
Comment (kirankumarkolli) [inline Directory.Build.props:3]: NEW API -> minor version revision
Hoptfix -> NO API changes and also scoped changes

File: changelog.md (hunk 2 above)
Comment (kirankumarkolli) [reply changelog.md:25]: Grat catch @sourabh1007 .

It violated semantic versioning. 
Covered in my comment above

**Pull request closed without merging**